agent/util-scripts/pbench-register-tool

```diff
@@ -278,6 +278,19 @@ else
 	fi
 fi
 
+invalids=0
+for (( i=0; ${i} < ${#remotes_A[@]}; i++ )); do
+	remote="${remotes_A[${i}]}"
+	validate-hostname "${remote}"
```

Comment:
Note that we don't use the `validate_remote` function from `base` because we want to report on all the remotes that are bad before exiting.